packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/android/challengeintegrationsteps.md

```diff
@@ -0,0 +1,53 @@
+### 1 - 4: Sign-in and Select Authenticator
+
+The challenge flow follows the same first four steps as the [enrollment flow](#integrate-sdk-for-authenticator-enrollment):
+
+- [Display the initial sign-in UI](#_1-display-the-initial-sign-in-ui).
+- [Start the sign-in flow](#_2-start-the-sign-in-flow).
+- [Authenticate the user credentials](#_3-authenticate-the-user-credentials).
+- [Display a list of authenticators available for enrollment](#_4-display-a-list-of-authenticators-available-for-enrollment).
+
+### 5: Display an OTP entry field
+
+The `CHALLENGE_AUTHENTICATOR` remediation type used for requesting a password is also used for one-time codes. A Google Authenticator code request is represented by an authenticator with a `type` of `IdxAuthenticator.Kind.APP` and a `key` of `"google_otp"`.
+
+This version of `handleChallenge()` includes Google Authenticator:
+
+```kotlin
+private suspend fun handleChallenge(remediation: IdxRemediation) {
+    // If no authenticators are found for challenge show error and abort
+    if (remediation.authenticators.size == 0) {
+        // handle error
+        return
+    }
+    val authenticator = remediation.authenticators.first()
+
+    when (authenticator.type) {
+        IdxAuthenticator.Kind.PASSWORD -> {
+            // Update the value in the remediation object and proceed to the next step in IDX flow
+            remediation["credentials.passcode"]?.value = password
+            remediation.proceed()
```

Comment:
This method doesn't exist.

Reply:
its an  extension function explained the the enrollment part which in the guide would appear before this section